plugins/firehydrant/src/components/ServiceDetailsCard/ServiceDetailsCard.tsx

```diff
@@ -16,12 +16,10 @@
 import React, { useEffect, useState } from 'react';
 import { DateTime } from 'luxon';
 import { ServiceAnalytics } from '../ServiceAnalytics/ServiceAnalytics';
-import {
-  Box,
-  Button as MaterialButton,
-  Typography,
-  makeStyles,
-} from '@material-ui/core';
+import Box from '@material-ui/core/Box';
+import MaterialButton from '@material-ui/core/MaterialButton';
```

Comment:
```suggestion
import Button from '@material-ui/core/Button';
```

I'm not sure why this was initially added using `Button as MaterialButton`. Can you make the suggestion and see what else might need to be changes as well? Thanks!